aspnetcore/release-notes/aspnetcore-9/includes/websockets-keep-alive-timeout.md

```diff
@@ -1,21 +1,18 @@
 ### Keep-Alive Timeout for WebSockets
 
-The [WebSockets middleware](https://learn.microsoft.com/aspnet/core/fundamentals/websockets#configure-the-middleware) can now be configured for keep alive timeouts.
+The [WebSockets middleware](https://learn.microsoft.com/aspnet/core/fundamentals/websockets#configure-the-middleware) can now be configured for keep-alive timeouts.
 
-The keep alive timeout will abort the WebSocket and throw from `WebSocket.ReceiveAsync` if a ping frame from the websocket protocol is sent by the server and the client doesn't reply with a pong frame within the specified timeout. The ping frame is automatically sent by the server and configured with `KeepAliveInterval`. This option is useful when wanting to detect connections that might be slow or ungracefully disconnected.
+The keep-alive timeout aborts the WebSocket connection and throws an exception from `WebSocket.ReceiveAsync` if both of the following conditions are met:
 
-The keep alive timeout can be configured globally for the WebSocket middleware:
-```csharp
-app.UseWebSockets(new WebSocketOptions { KeepAliveInterval = TimeSpan.FromSeconds(15) });
-```
+* The server sends a ping frame using the websocket protocol.
+* The client doesn't reply with a pong frame within the specified timeout.
+
+The server automatically sends the ping frame and configures it with `KeepAliveInterval`. This option is useful for detecting connections that might be slow or ungracefully disconnected.
```

Comment:
Why is this called an "option"?

Reply:
"This feature ..." sounds good I think.
Or "The keep-alive timeout setting ...", since we just mentioned `KeepAliveInterval` in the previous sentence but are referring to the timeout.